src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -153,11 +153,11 @@ protected override sealed Symbol OriginalSymbolDefinition
         /// (for example, interfaces), null is returned. Also the special class System.Object
         /// always has a BaseType of null.
         /// </summary>
-        internal abstract NamedTypeSymbol BaseTypeNoUseSiteDiagnostics { get; }
```

Comment:
Consider keeping the existing property since most callers don't use the new argument. Then the new argument could be non-optional.
```
internal NamedTypeSymbol BaseTypeNoUseSiteDiagnostics =>
    GetBaseTypeNoUseSiteDiagnostics(ignoreNonNullTypesAttribute: false);
```

Reply:
I've removed the default parameter value and implemented Chuck's suggestion. The callers that don't care about this parameter go through a simple property as before.

---
In reply to: [200711324](https://github.com/dotnet/roslyn/pull/28320#discussion_r200711324) [](ancestors = 200711324)